cypress/e2e/filter/filterChips.cy.ts

```diff
@@ -6,6 +6,17 @@ export function removeFilterChip() {
   cy.get(".moj-filter__tag").should("not.exist")
 }
 
+export function filterByDateRange(dateRangeId: string) {
+  cy.get("#date-range").click()
+  cy.get(dateRangeId).click()
+}
+
+function filterByCustomDateRange(dateFrom: string, dateTo: string) {
```

Comment:
I think in most cases, we prefer using arrow functions over "traditional functions", while in some cases, it just a style choice. For instance I like arrow functions because they have a more concise syntax and quite readable. But there are technical differences too:
- const prevents reassignment but `function` not
- arrow functions has to be declared before calling it, while functions can be defined after calling them

nit picking is over :)

Reply:
It makes sense. I've updated!